app/emails/dreamail.py

```diff
@@ -0,0 +1,87 @@
+from flask import current_app
+from .util import render_email_template, send_or_handle_error, fill_template
+
+
+def send_dreamail():
+    from app.api.services import (
+        audit_service,
+        audit_types,
+        suppliers
+    )
+
+    result = suppliers.get_suppliers_with_rejected_price()
+
+    for item in result:
+        supplier_code = item['code']
+        supplier = suppliers.find(code=item['code']).one_or_none()
+
+        sent = audit_service.find(
+            object_id=supplier.id,
+            object_type='Supplier',
+            type='seller_to_review_pricing_case_study_email'
+        ).one_or_none()
+        if sent:
+            continue
+
+        case_studies = supplier.case_studies
+        domains = supplier.domains
+
+        option_1_aoe = []
+        option_2_aoe = []
+        for domain in domains:
+            domain_name = domain.domain.name
+            case_studies_in_domain = [cs for cs in case_studies if cs.data['service'] == domain_name]
+            no_approved_case_studies = all(cs.status == 'rejected' for cs in case_studies_in_domain)
+            if no_approved_case_studies:
+                option_2_aoe.append('* {}'.format(domain_name))
+            else:
+                option_1_aoe.append('* {}'.format(domain_name))
+
+        dreamail_option_1_content = ''
+        if option_1_aoe:
+            dreamail_option_1_content = fill_template(
+                'dreamail_option_1.md',
+                aoe='\n'.join(option_1_aoe)
+            )
+
+        dreamail_option_2_content = ''
+        if option_2_aoe:
+            dreamail_option_2_content = fill_template(
+                'dreamail_option_2.md',
+                frontend_url=current_app.config['FRONTEND_ADDRESS'],
```

Comment:
Does this template need `frontend_url`?

Reply:
used here: [marketplace.service.gov.au]({{ frontend_url }})